inc/customizer/controls/layouts.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+class Primer_Customizer_Layouts_Control extends WP_Customize_Control {
+
+	/**
+	 * Enqueue some custom css for this control
```

Comment:
@jonathanbardo Sorry just caught this. Please put a period (`.`) at the end of your descriptions and use `@action` and `@since`.